python/adelphi/tests/integration/test_diff.py

```diff
@@ -0,0 +1,23 @@
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+
+from tests.integration import DockerSchemaTestMixin, AdelphiExportMixin
+
+class TestDiff(unittest.TestCase, DockerSchemaTestMixin, AdelphiExportMixin):
+
+    def setUp(self):
+        super(TestDiff, self).setUp()
+
+
+    def getExportCommand(self):
+        return "export-cql --no-metadata"
+
+
+    def getSchemaPath(self):
+        return "tests/integration/resources/diff-base-schema.cql"
```

Comment:
Since we're in Python land now, can't we call the function in `schema_util.py` to generate the base schema file on the fly instead of having to commit it as a text file?
Another benefit would be not having to remember to update that CQL file when changing the schema generator.

Reply:
Hmmmm, we might be able to do this for the diff test but not for the nosqlbench test (at least not yet) due to the brittleness of the current impl w.r.t. types.  Extending the schema_util impl to optionally add things based on params or a definition of some kind would probably solve that but I'm not sure that's something we need right now; the file-based approach does work in the short-term.

Maybe filing another ticket to expand the schema_util impl is the right way to go here.

I'll take a quick run at this and see how bad it is to change for the diff test.

Man, we also really ought to think about changing the name of that test... calling it a "diff" test is (a) wrong now and (b) telling us more about the impl of the test rather than what it's testing even if it were right.

Reply:
After https://github.com/datastax/adelphi/pull/104/commits/28232ade777bb9af2863c311aa148cf08c9248c2 and https://github.com/datastax/adelphi/pull/104/commits/808e4b00798276b898ef10da80630507a5e64d89 this should be working now.